contracts/RewardPool.vy

```diff
@@ -1,29 +1,23 @@
-# @version 0.2.16
+# @version 0.3.4
 """
-@title Curve Fee Distribution
-@author Curve Finance
+@title YFI Reward Pool
+@author Curve Finance, Yearn Finance
 @license MIT
 """
-
 from vyper.interfaces import ERC20
 
-
-interface VotingEscrow:
+interface VotingYFI:
     def user_point_epoch(addr: address) -> uint256: view
     def epoch() -> uint256: view
     def user_point_history(addr: address, loc: uint256) -> Point: view
     def point_history(loc: uint256) -> Point: view
     def checkpoint(): nonpayable
-    def deposit_for(addr: address, amount: uint256): nonpayable
-
-event CommitAdmin:
-    admin: address
+    def token() -> address: view
```

Comment:
by the way -- `token()` can return an interface, e.g. `token() -> ERC20`. it reduces a bit of boilerplate casting raw addresses to interfaces, and is slightly more typesafe

Reply:
fixed in https://github.com/yearn/veYFI/pull/169